lib/obs_github_deployments/deployment.rb

```diff
@@ -72,7 +72,9 @@ def latest_status
     end
 
     def create
-      options = { auto_merge: false }
+      # auto_merge: Do not merge the default branch into the requested deployment branch if necessary
+      # required_contexts: Do not verify if CI status checks passed on the branch to be deployed. This allows us from deploying if, for example, somehow codecov failed.
```

Comment:
```suggestion
      # required_contexts: Do not verify if CI status checks passed on the master branch. This allows to create a deployment, even if somehow codecov failed.
```

Reply:
Whoopsie, that went wrong when I changed the sentence. I'll fix it!